source/Scrapers/TeamCityBuildArtifactScraper.cs

```diff
@@ -27,10 +27,11 @@ public TeamCityBuildArtifactScraper(IMetricFactory metricFactory, IConfiguration
         protected override async Task Scrape(CancellationToken stoppingToken)
         {
             await Task.CompletedTask;
-            
+
             var teamCityToken = configuration.GetValue<string>("TEAMCITY_TOKEN");
             var teamCityUrl = configuration.GetValue<string>("BUILD_SERVER_URL");
-            var teamCityClient = new TeamCityClient(teamCityUrl, true);
+            bool useSSL = configuration.GetValue<bool>("USE_SSL");
```

Comment:
Nit (for consistency):
```suggestion
            var useSSL = configuration.GetValue<bool>("USE_SSL");
```
Because you're passing an explicit type to `GetValue<>()` the compiler knows that the variable will be a `bool`. And it's idiomatic to use `var` instead of explicit types, for the most part.

Reply:
Same suggestion on the others.